CHANGE_LOG.md

```diff
@@ -13,4 +13,9 @@ All notable changes to this project will be documented in this file.## [Unreleas
 
 ## [0.1.1] - 2022-05-20
```

Comment:
Changelog should look like this 

```
# Changelog
All notable changes to this project will be documented in this file.

[Unreleased]
### Added
- Added support for PostChat Link Mode
- Added more telemetry and broadcast events for proactive chat pane
- Added the flag for toggling the focus on chatbutton component
- Updated the version of @microsoft/omnichannel-chat-components to @0.1.0-main.3e712be
```

Until we official release a version `v0.1.0`, which will become

```
# Changelog
All notable changes to this project will be documented in this file.

[Unreleased]

[1.0.0] - YYYY-MM-DD
### Added
- Added support for PostChat Link Mode
- Added more telemetry and broadcast events for proactive chat pane
- Added the flag for toggling the focus on chatbutton component
- Updated the version of @microsoft/omnichannel-chat-components to @0.1.0-main.3e712be
```

Reply:
Reference: https://keepachangelog.com/en/1.0.0/